CMakeLists.txt

```diff
@@ -97,12 +100,16 @@ target_link_libraries(clio_server PUBLIC clio)
 if(BUILD_TESTS)
   set(TEST_TARGET clio_tests)
   add_executable(${TEST_TARGET}
-  unittests/RPCErrors.cpp
-  unittests/Backend.cpp
-  unittests/Logger.cpp
-  unittests/Config.cpp
-  unittests/ProfilerTest.cpp
-  unittests/DOSGuard.cpp)
+    unittests/Playground.cpp
```

Comment:
don't forget this one

Reply:
i did not forget it - playground.cpp as it is now should be in the project (otherwise you would have to write your own each time). the point is not to commit the tmp stuff into it later on :)